store/storetest/group_supplier.go

```diff
@@ -1169,15 +1169,15 @@ func testPendingAutoAddChannelMembers(t *testing.T, ss store.Store) {
 	require.Len(t, res.Data, 1)
 
 	// No result if Channel deleted
-	res = <-ss.Channel().Delete(channel.Id, model.GetMillis())
-	require.Nil(t, res.Err)
+	err := ss.Channel().Delete(channel.Id, model.GetMillis())
+	require.Nil(t, err)
```

Comment:
could we add messages here? (to maintain consistency with what you've done above)

Reply:
I'm ok with this proposal, but we are not very consistent with this thing, and actually, I think is more common to use the `require.Nil(t, err)` without the explanation.

Reply:
Okay, I'm fine with as-is then.